words/words.md

```diff
@@ -338,9 +338,9 @@ Was also tun? Einen Lockdown nach dem anderen?
 
 Diese Option wurde zuerst im Bericht des Imperial College vom 16. März und später erneut in einem Harvard-Papier vorgeschlagen [^lockdown_harvard].
 
-[^lockdown_harvard]: In Abwesenheit anderer Interventionen ist das Überschreiten der Intensivpflege-Kapazitäten eine Schlüsselgröße für den Erfolg des social distancing. Um dies zu vermeiden, kann längeres oder zeitweiliges social distancing bis ins Jahr 2022 erforderlich sein. (übersetzt): “Absent other interventions, a key metric for the success of social distancing is whether critical care capacities are exceeded. To avoid this, prolonged or intermittent social distancing may be necessary into 2022.”[Kissler und Tedijanto et al] (https://science.sciencemag.org/content/early/2020/04/14/science.abb5793)
+[^lockdown_harvard]: In Abwesenheit anderer Maßnahmen ist eine Schlüsselmetrik für den Erfolg der sozialen Distanzierung, ob die Kapazitäten der Intensivpflege überschritten werden. Um dies zu vermeiden, kann eine längere oder zeitweilige soziale Distanzierung bis ins Jahr 2022 erforderlich sein. [Kissler und Tedijanto et al] (https://science.sciencemag.org/content/early/2020/04/14/science.abb5793)
```

Comment:
```suggestion
[^lockdown_harvard]: In Abwesenheit anderer Maßnahmen ist eine Schlüsselmetrik für den Erfolg der sozialen Distanzierung, ob die Kapazitäten der Intensivstationen überschritten werden. Um dies zu vermeiden, kann eine längere oder periodische räumliche Distanzierung bis ins Jahr 2022 erforderlich sein. (übersetzt): “Absent other interventions, a key metric for the success of social distancing is whether critical care capacities are exceeded. To avoid this, prolonged or intermittent social distancing may be necessary into 2022.” [Kissler und Tedijanto et al] (https://science.sciencemag.org/content/early/2020/04/14/science.abb5793)
```
Wir hatten uns darauf geeinigt Originalzitate mit Übersetzung zu liefern. Wie Originalzitat und Übersetzung formatiert sind, würde ich nach den PRs noch mal vereinheitlichen.
Bei social distancing bin ich mir unsicher. Bei Wikipedia bekommt man da gerade "räumliche Distanzierung" was mir eigentlich besser gefällt (da es genauer beschreibt was gemacht wird) als das Original.

Reply:
Ob "Intensivpflege" oder "Intensivstationen" ist mir eigentlich egal